lib/iex/lib/iex/introspection.ex

```diff
@@ -115,13 +126,42 @@ defmodule IEx.Introspection do
         end
         :ok
       else
-        :not_found
+        if has_callback?(mod, fun, arity) do
+          :behaviour_found
+        else
+          :not_found
+        end
       end
     else
       :no_docs
     end
   end
 
+  defp has_callback?(for, fun) do
```

Comment:
I changed it here anyway since `for` is currently only used in the functions that take care of printing the final result, so `has_callback?` is now more consistent with the code that is around it.